config/routes.rb

```diff
@@ -1,3 +1,3 @@
-ShiftCommerce::Engine.routes.draw do
+Rails.application.routes.draw do
```

Comment:
This applies the routes to the parent application, which gives us access to variables in the parent application controller, and global variables such as session. This also ensures that parent styles can be applied to views in gem-provided controllers.